jobs/JGLOBAL_AERO_ANALYSIS_INITIALIZE

```diff
@@ -19,13 +19,15 @@ GDUMP="gdas"
 ##############################################
 
 # Generate COM variables from templates
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_CHEM_ANALYSIS
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COM_OBS:COM_OBS_TMPL \
```

Comment:
```suggestion
    COMIN_OBS:COM_OBS_TMPL \
```

Reply:
I purposefully left out COM_OBS (for now) since I'll need to change a bunch of YAMLs/Jinja templates as part of that. Do you want that in this PR, or a subsequent PR?

Reply:
It can be deferred.